test/helpers/versions.js

```diff
@@ -0,0 +1,29 @@
+var helpers = require('./index');
+var massive = require('../../index');
+var connectionString = helpers.connectionString;
+var gte95;
+
+function init() {
+  var db = massive.connectSync({
+    connectionString: connectionString
+  });
+  var res = db.runSync('SELECT version()');
+  var version = res[0].version;
+  var matches = version.match(/PostgreSQL (\d+)\.(\d+)/);
```

Comment:
I'm not too familiar with Postgres version strings, so not sure how fragile this pattern is. Just matching the major and minor version numbers and ignoring the rest.